editor/editor_node.cpp

```diff
@@ -7171,16 +7171,25 @@ EditorNode::EditorNode() {
 	dock_select->set_v_size_flags(Control::SIZE_EXPAND_FILL);
 	dock_vb->add_child(dock_select);
 
-	if (!SceneTree::get_singleton()->get_root()->is_embedding_subwindows() && !EDITOR_GET("interface/editor/single_window_mode") && EDITOR_GET("interface/multi_window/enable")) {
-		dock_float = memnew(Button);
-		dock_float->set_icon(theme->get_icon("MakeFloating", EditorStringName(EditorIcons)));
-		dock_float->set_text(TTR("Make Floating"));
-		dock_float->set_focus_mode(Control::FOCUS_NONE);
-		dock_float->set_h_size_flags(Control::SIZE_SHRINK_CENTER);
-		dock_float->connect("pressed", callable_mp(this, &EditorNode::_dock_make_selected_float));
-
-		dock_vb->add_child(dock_float);
-	}
+	dock_float = memnew(Button);
+	dock_float->set_icon(theme->get_icon("MakeFloating", EditorStringName(EditorIcons)));
+	dock_float->set_text(TTR("Make Floating"));
+	const bool multi_window_enabled = !SceneTree::get_singleton()->get_root()->is_embedding_subwindows() && !EDITOR_GET("interface/editor/single_window_mode") && EDITOR_GET("interface/multi_window/enable");
```

Comment:
This PR adds 2 more instances that uses this to confirm multi-window is enabled, with the total now being 4[^1]. We really should add a centralized way to ask this that checks those conditions by itself.

[^1]: The other two instances being in https://github.com/godotengine/godot/blob/b7449a3d1aebd34026c7a2279a1b18ff0f02ccfa/editor/editor_node.cpp#L7174 and https://github.com/godotengine/godot/blob/b7449a3d1aebd34026c7a2279a1b18ff0f02ccfa/editor/window_wrapper.cpp#L317.